homeassistant/components/motioneye/__init__.py

```diff
@@ -111,7 +111,7 @@ def _add_camera(
 
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
-        identifiers={device_identifier},
+        identifiers={device_identifier},  # type: ignore[arg-type]
```

Comment:
This looks related to discussion here: https://github.com/home-assistant/core/pull/49670/files#r619862785

Reply:
Yes, I spotted that. 

I fear future heartache from not using a (str, str) device identifier, so let me see if I can get https://github.com/home-assistant/core/pull/49774/files in first ...

Reply:
That is now committed, and this PR is now adapted. Thanks for the pointer to the discussion.